.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+<!--
+  See the contributing guide for detailed guidance about contributing.
+  https://github.com/perses/perses/blob/main/CONTRIBUTING.md
+ -->
+
+# Checklist
+
+- [ ] Pull request has a descriptive title and context useful to a reviewer.
+- [ ] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
+- [ ] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).
```

Comment:
Not sure if we really need this here since there's a github check, but thought it might be helpful as a pointer for people who are new and may not be sure what that github check is doing.

Reply:
I think this is a good call -- it is one of the main things to remember and if I was a new contributor, I would much rather see this first than run into the DCO warning (which can be harder to debug)